src/renderer/containers/main/search.ts

```diff
@@ -25,9 +25,11 @@ class Search extends Container<SearchState, MainCTX> {
 
   _isNoteMatch = ( note: NoteObj, query: string ): boolean => {
 
+    const queryRe = new RegExp ( _.escapeRegExp ( query ), 'ig' );
```

Comment:
This is not good, creating the regex for each note is expensive.

Reply:
But most importantly, did you benchmark this against converting the content and query strings to lowercase?

Reply:
@fabiospampinato Regex is faster in this case. You can see for yourself here: https://jsperf.com/match-vs-indexof-tolowercase/2

Reply:
@tylerdiaz maybe pass it as third parameter, we need the string for fuzzy matching the title.

Reply:
@fabiospampinato 👍

Reply:
@tylerdiaz I've run the benchmark but I don't think it's properly done, the regexes are re-created after each iteration. Maybe that's why indexOf/toLowerCase is 100x faster for me in this benchmark.

Reply:
@fabiospampinato That's interesting, I see your benchmark results on Chrome are drastically different than mine on Firefox. Let me address the test and try on Chrome.

Reply:
@fabiospampinato New test here which only initializes the regex once. https://jsperf.com/regex-vs-indexof-tolowercase On Chrome 73, Regex is 98% slower.

Reply:
The regex is like 50x slower on Chrome it seems, impressive. You should test on chrome as we are using its rendering engine under the hood. Let's stick with indexOf then.

Reply:
@tylerdiaz since the regex-based solution is way slower can you revert to `indexOf` and `toLowerCase` so that I can merge this?

Reply:
@fabiospampinato Updated

Reply:
@fabiospampinato 👍 Done